### DISCUSSION TITLE
Error on authorisation

### DISCUSSION THREAD

**Discussion by Patrick-UK** [alexkn/node-red-contrib-homeconnect#58] [Q&A] [answered]
When setting up the connection, during the authorisation step I get a 404 Not Found error.
In the API docs it stated image not found, but no other details.

Any ideas what might cause this?

**Comment by alexkn** [comment]
I don't get where exactly you get the error.
Can you explain more what you are doing or add a screenshot?

**Reply by Patrick-UK** [reply]
When I am selecting the start authorising button and going through that step.
I get to login and approve, but then it returns 404. Screenshots below. Thanks

![image](https://user-images.githubusercontent.com/86563655/158058043-e252e917-15ab-48a6-88e9-b27b4c727b62.png)
![image](https://user-images.githubusercontent.com/86563655/158058104-59758117-0850-4bfb-96ee-f5ee91d28dac.png)
![image](https://user-images.githubusercontent.com/86563655/158058122-de163ff3-6f38-48d4-8f79-489015f6c4c3.png)

**Comment by rempam** [comment]
Same problem for me.

IT is Home Assistent relaties. No problem with Node Red on a Windows PC. No

**Comment by Patrick-UK** [comment, marked as answer]
Finally got it to work! Had to open Node-Red directly in another tab using the port 1880. Then it authorised. 
I found the issue could be related to opening Node-Red within Home Assistant as it places it within an iframe and this seems to be causing the it to not to find the ingress completely. There is a few issues noted about this for other add-ins so thought I would give it a try.

Hopefully this will help others with a similar issue.

**Reply by rempam** [reply]
thanks! Works for me too
